working-examples/reflow-carousel-panel-horizontal-scroll/index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>Carousel panels that scroll horizontally are designed to fit within a width of 320 CSS pixels on a vertically scrolling page</title>
+        <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
+        <style>
+            * {
+                margin: 0;
+                padding: 0;
+                box-sizing: border-box;
+            }
+
+            body {
+                font-family: sans-serif;
+                margin: 0;
+                padding: 0;
+                color: #000;
+            }
+
+            .carousel {
+                position: relative;
+                overflow: hidden;
+            }
+
+            h2 {
+              margin: 20px;
+            }
+            .carousel .prod-list > ul {
```

Comment:
We could make the scrolling smoother by adding: `scroll-snap-type: x mandatory` in this block and `scroll-snap-align: center` in the `.carousel .prod-list > ul li` block.